From ffea9fde3848badb0c17ee7ff68069e95c7764b1 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Sat, 12 Jul 2014 16:37:02 +0200
Subject: [PATCH 39/41] DOC: mention that "compression offload" is ignored in
 defaults section

This one is not inherited from defaults into frontends nor backends
because it would create a confusion situation where it would be hard
to disable it (since both frontend and backend would enable it).
---
 doc/configuration.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/doc/configuration.txt b/doc/configuration.txt
index 96da1cc..d69bb11 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -2158,7 +2158,8 @@ compression offload
   invalid payloads. In this case, simply removing the header in the
   configuration does not work because it applies before the header is parsed,
   so that prevents haproxy from compressing. The "offload" setting should
-  then be used for such scenarios.
+  then be used for such scenarios. Note: for now, the "offload" setting is
+  ignored when set in a defaults section.
 
   Compression is disabled when:
     * the request does not advertise a supported compression algorithm in the
-- 
1.8.5.5

